server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java

```diff
@@ -120,6 +120,7 @@ protected void doInit(JobContext context) {
         this.result.setRegion(JobPropertiesUtils.getRegionName(jobProperties));
         this.result.setCloudProvider(JobPropertiesUtils.getCloudProvider(jobProperties));
         this.connectionSession = generateSession();
+        ConnectionSessionUtil.getSqlCommentProcessor(connectionSession).setDelimiter(parameters.getDelimiter());
```

Comment:
should set delimiter while construct CommentProcessor in `generateSession` method.